test_helpers/main.go

```diff
@@ -190,6 +194,7 @@ func StartTarantool(startOpts StartOpts) (TarantoolInstance, error) {
 		os.Environ(),
 		fmt.Sprintf("TEST_TNT_WORK_DIR=%s", startOpts.WorkDir),
 		fmt.Sprintf("TEST_TNT_LISTEN=%s", startOpts.Listen),
+		fmt.Sprintf("TT_MEMTX_USE_MVCC_ENGINE=%t", startOpts.MemtxUseMvccEngine),
```

Comment:
Now we use custom `TEST_TNT_` vars to configure `box.cfg`. I think it's better too use them for consistency.

Reply:
Done.